example-config/etesync.ini

```diff
@@ -0,0 +1,18 @@
+# uwsgi configuration file
+# typical location of this file would be /etc/uwsgi/sites/etesync.ini
+
+[uwsgi]
+folder = /path/to/etesync
+
+socket = /run/uwsgi/etesync.sock
```

Comment:
maybe better to call it etesync_server.sock to be consistent and just make it a var then (like you did with folder).
Btw, you don't need to have both folder and chdir. You can just set chdir and use it again in home.